### PR TITLE
Notification about cookies added

### PR DESCRIPTION
**Pull request by Dzeranov**
Cookies notification added.
Expiration time is 30 days.
https://app.asana.com/0/1200854944775310/1201707522540591

### CODE REVIEW

**Comment by vkomodey** [comment]
Could you please show how is it look like?

**Comment by Dzeranov** [comment]
> Could you please show how is it look like?

image

**Comment by vkomodey** [comment]
please be careful. the target branch should be master

File: src/App.jsx [hunk 1]
```diff
@@ -58,6 +59,21 @@ const App = () => {
...
+        location="bottom"
+        buttonClasses="btn btn-custom form-control"
+        buttonText="I understand"
+        cookieName="cookiesConsent"
+        style={{ background: 'rgba(50, 10, 141, 0.8)' }}
+        expires={30}
+      >
+        We use cookies. By using this site, you consent to our{' '}
+        <span>
+          <a href="https://humanprotocol.org/app/privacy-policy" style={{ fontSize: '14px' }}>
```
Comment (vkomodey) [inline src/App.jsx:72]: For a better UX please add attributes `target="_blank"` and `rel="noreferrer noopener"` for opening the privacy policy in the new page

**Review by vkomodey — CHANGES_REQUESTED** [review]
All good. Last thing about privacy policy href and we are gtg

**Pull request merged**